_layouts/index.html

```diff
@@ -17,7 +17,14 @@
       {% if signature.affiliation %}
       , {{ signature.affiliation }}
       {% endif %}
+
+      {% if signature.title %}
+      , {{ signature.title }}
+      {% endif %}
     </li>
+    {% if signature.github %}
+    <!-- Github user: {{ signature.github }} -->
+    {% endif %}
```

Comment:
Can we do title, then affiliation? People see to be preferring that in their signatures.